airflow-core/src/airflow/decorators/__init__.py

```diff
@@ -41,15 +32,6 @@
     "dag",
     "task",
     "task_group",
-    "python_task",
-    "virtualenv_task",
-    "external_python_task",
-    "branch_task",
-    "branch_virtualenv_task",
-    "branch_external_python_task",
-    "short_circuit_task",
-    "sensor_task",
-    "bash_task",
```

Comment:
We should ensure that we add a backcompat layer for these

Reply:
The point of interaction will remain as `@task` which is in `airflow.decorators` right? Wouldnt it be the same?

Reply:
aah yes, should be

Reply:
It will need a compat layer when i follow this up and move it to task sdk.